Tests/Realm.Tests/Database/ObjectSchemaTests.cs

```diff
@@ -884,7 +891,8 @@ public void ObjectSchema_GetBuilder_ContainsAllProperties()
 
             Assert.That(originalSchema, Is.EquivalentTo(builder));
             Assert.That(originalSchema.Name, Is.EqualTo(builder.Name));
-            Assert.That(originalSchema.IsEmbedded, Is.EqualTo(builder.IsEmbedded));
+            Assert.That(originalSchema.ObjectType == ObjectSchema.ObjectSchemaType.EmbeddedObject,
+                Is.EqualTo(builder.RealmSchemaType == ObjectSchema.ObjectSchemaType.EmbeddedObject));
```

Comment:
This seems like a weird check - why not just assert that `originalSchema.ObjectType, Is.EqualTo(builder.RealmSchemaType)`?